scripts/Tools/Makefile

```diff
@@ -124,8 +124,6 @@ endif
 
 ifeq ($(strip $(PIO_VERSION)),1)
   CPPDEFS += -DPIO1
-else
```

Comment:
This line is not needed for our pio2, I think that it may be needed for scorpio - @jayeshkrishna can you confirm.

Reply:
Yes, scorpio needs this line.

Reply:
@jayeshkrishna  I believe that you can add it in the scorpio buildlib script instead of here.  Can you please confirm?

Reply:
Nevermind, I have a fix that should work for both of us.

Reply:
Thanks, the changes look good to me. I will try an E3SM build today with the changes and approve the PR.